### DISCUSSION TITLE
How can I create an accessible bot menu?

### DISCUSSION THREAD

**Discussion by cverster** [def-studio/telegraph#435] [Q&A] [answered]
I've tried to register commands on my bot (which I did successfully), but no bot menu appears?

How do I get an accessible bot menu that users can see in Telegram? (the blue menu that pops up next to your message input bar)

**Comment by fabio-ivona** [comment, marked as answer]
Hi, please review Telegram Bot's [documentation](https://core.telegram.org/bots/features#commands)
